cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/TOOLCHAIN_GCC/TARGET_M0/irq_cm0.S

```diff
@@ -18,17 +18,21 @@
  * -----------------------------------------------------------------------------
  *
  * Project:     CMSIS-RTOS RTX
- * Title:       Cortex-M0 Exception handlers
+ * Title:       ARMv6-M Exception handlers
  *
  * -----------------------------------------------------------------------------
  */
 
 
         .syntax  unified
 
+        #include "rtx_def.h"
```

Comment:
this is causing a problem as we somehow get some C/C++ headers to this .S file and it just errors with strange errors. 

We ignored the inclusion previously because of this, will need to check how to fix it this time.